Dockerfile

```diff
@@ -25,15 +25,6 @@ RUN apk add --no-cache \
         py3-wheel \
     && pip3 install --no-cache-dir --requirement /tmp/requirements.txt \
     && rm -rf /root/.cache
-# Managing the below ansible dependencies is covered in docs/dev/ansible-modules.md
-RUN mkdir -p /usr/share/ansible/collections \
```

Comment:
Any concerns with drifting versions for either Ansible or these collections?

Reply:
version skew with ansible-core might be issue. We have this is a problem with current configuration as well. 

Documentation for each of these plugins only states that they are compatible with ansible core version X or higher. ex. https://docs.ansible.com/ansible/latest/collections/community/general/index.html
I think we can either
(1) add version constraint for ansible collections using https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#installing-an-older-version-of-a-collection
I check dependabot does not maintain galaxy collection bumps as of now. https://github.com/dependabot/dependabot-core/issues/3522 . we will have to maintain it manually.
We have to make sure that they are compatible with `ansible-core` version in `requirements.txt` file.
add a note in `requirements.txt` to ensure to check collection's versions when bumping `ansible-core` version.

(2) If we should always install latest version of the collections by installing collection every time we build KIB image instead of copying it from cached devkit image.
revert this change and remove installing collections in devkit image. 

wdyt?

Reply:
Sorry, I should have been clearer. I was more wondering about new version skew between kib image and devkit image?
It sounds like this PR is adding to that.